### DISCUSSION TITLE
Fail within compiling, need help!

### DISCUSSION THREAD

**Discussion by EagleEi** [arendst/Tasmota#18648] [Q&A]
Hey all,
I’m use a lot of Tasmota devices all standard sensor.bin, I love it!
Now I will use a Esp32 with the sensors for BME and BMP also inclue the VELM7700 + Bluetooth for HA BLE bridge too. 
I tried a lot of compiling trials, with PlatformIO and GitPod etc. but so pitty I always failed, the most points I don’t understand and after many hours trial and errors I have to give up.

I’m looking for someone who can help we to compile me this tasmota-special.bin
I’m sure this is very easy for someone who knows how it works.

is there anybody out there for helping me?
Thank you in advance!

Best regards
Benny

open-diy-projects.com

**Comment by SimonFili** [comment]
Maybe if you list the steps and modifications you did, that could help?
